### PR TITLE
Add TotW #55

### PR DESCRIPTION
**Pull request by manshreck**
Also, add note styling to Bazel usage.

### CODE REVIEW

**Comment by manshreck** [comment]
Note: second "code sample" is prettyprinting even though I don't want it to. Filed an issue for this.

https://github.com/abseil/abseil.github.io/issues/80

File: _posts/2017-10-20-totw-55.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+type: markdown
+---
+
+Originally posted Sep 12, 2013
+
+*by Titus Winters [(titus@google.com)](mailto:titus@google.com)*
+
+Updated Oct 20, 2017
+
+Quicklink: **abseil.io/tips/55**
```
Comment (tituswinters) [inline _posts/2017-10-20-totw-55.md]: This doesn't seem to linkify

File: _posts/2017-10-20-totw-55.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+names for a `std::unique_ptr`: one is OK, but multiple names for the same
+pointer value are not.
+
+Let’s count some names. At each line number, count the number of names alive at
+that point (whether in scope or not) that refer to a `std::unique_ptr`
+containing the same pointer. If you find any line with more than one name for
+the same pointer value, that’s an error!
+
+```c++
+ std::unique_ptr<Foo> NewFoo() {
```
Comment (tituswinters) [inline _posts/2017-10-20-totw-55.md]: this is indented one space and the rest of the example isn't

Comment (manshreck) [reply _posts/2017-10-20-totw-55.md]: Fixed.

File: _posts/2017-10-20-totw-55.md (hunk 1 above)
Comment (manshreck) [reply _posts/2017-10-20-totw-55.md]: GH Pages doesn't like autolinking to same document. I added a manual link to self.

**Pull request merged**